deploy/overlays/dev-template/config.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     disableSSLVerification: ${GIT_DISABLE_SSL_VERIFICATION}
   requireArtifactVerification: true
   buildSettings:
-    buildRequestCPU: "2"
+    buildRequestCPU: "300m"
+    buildLimitCPU: "4"
```

Comment:
Do our dev clusters have 4 capability (especially with other things running)?

Reply:
It should not matter for the limit, our dev cluster have 4 cores so this will let it use as much as possible. Even if other things are running it won't matter.